From weiyi.huang@gmail.com  Mon Oct 20 13:58:51 2008
From: Huang Weiyi <weiyi.huang@gmail.com>
Date: Sat, 18 Oct 2008 11:53:07 +0800
Subject: staging: wlan-ng: prism2: remove more duplicated #include
To: gregkh@suse.de
Cc: linux-kernel@vger.kernel.org
Message-ID: <20081018114621.61C3.WEIYI.HUANG@gmail.com>


Removed duplicated #include "version.h" in
drivers/staging/wlan-ng/p80211wep.c.

Signed-off-by: Huang Weiyi <weiyi.huang@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/wlan-ng/p80211wep.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/staging/wlan-ng/p80211wep.c
+++ b/drivers/staging/wlan-ng/p80211wep.c
@@ -64,7 +64,6 @@
 /*================================================================*/
 /* Project Includes */
 
-#include "version.h"
 #include "p80211hdr.h"
 #include "p80211types.h"
 #include "p80211msg.h"
